webdriver/tests/support/fixtures_bidi.py

```diff
@@ -20,11 +20,12 @@
 async def add_preload_script(bidi_session):
     preload_scripts_ids = []
 
-    async def add_preload_script(function_declaration, arguments=None, sandbox=None):
+    async def add_preload_script(function_declaration, arguments=None, contexts=None, sandbox=None):
         script = await bidi_session.script.add_preload_script(
             function_declaration=function_declaration,
             arguments=arguments,
             sandbox=sandbox,
+            contexts=contexts,
```

Comment:
nit: align arg order
```suggestion
            contexts=contexts,
            sandbox=sandbox,
```